### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, open 'undefined'

### ISSUE THREAD

**Issue by Globegitter** [v3ss0n/goto#72]
[Enter steps to reproduce below:]
1. Just tried to go to the Declaration of a relatively imported ES6 module

**Atom Version**: 1.3.0-beta4
**System**: Ubuntu 14.04.3
**Thrown From**: [goto](https://github.com/v3ss0n/goto) package, v1.7.1
### Stack Trace

Uncaught Error: ENOENT: no such file or directory, open 'undefined'

```
At fs.js:549

Error: ENOENT: no such file or directory, open 'undefined'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.module.(anonymous function) [as openSync] (ATOM_SHELL_ASAR.js:137:20)
    at Object.fs.readFileSync (fs.js:397:15)
    at Object.fs.readFileSync (ATOM_SHELL_ASAR.js:385:29)
    at SymbolIndex.module.exports.SymbolIndex.processFile (/home/mpadourek/.atom/packages/goto/lib/symbol-index.coffee:182:15)
    at SymbolIndex.module.exports.SymbolIndex.update (/home/mpadourek/.atom/packages/goto/lib/symbol-index.coffee:113:12)
    at SymbolIndex.module.exports.SymbolIndex.gotoDeclaration (/home/mpadourek/.atom/packages/goto/lib/symbol-index.coffee:129:6)
    at Object.module.exports.gotoDeclaration (/home/mpadourek/.atom/packages/goto/lib/index.coffee:36:22)
    at atom-workspace.atom.commands.add.goto:declaration (/home/mpadourek/.atom/packages/goto/lib/index.coffee:22:31)
```
### Commands

```
     -3:38.3.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -3:36.7.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -3:35.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -3:35.3.0 goto:declaration (atom-text-editor.editor)
     -3:22.8.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -3:20.2.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -3:20 core:confirm (atom-text-editor.editor.mini.is-focused)
     -3:20 goto:declaration (atom-text-editor.editor)
     -1:40.9.0 command-palette:toggle (atom-text-editor.editor.is-focused)
  2x -1:37.8.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -1:37.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:37.4.0 goto:rebuild-index (atom-text-editor.editor)
     -0:03.7.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:01.8.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:01.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:01.6.0 goto:declaration (atom-text-editor.editor)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "base16-tomorrow-dark-theme"
    ],
    "disabledPackages": [
      "symbols-view"
    ]
  },
  "goto": {
    "moreIgnoredNames": "node_modules, cordova_out, plz-out, .idea, .git, .plz-cache"
  }
}
```
### Installed Packages

``` coffee
# User
goto, v1.7.1
pigments, v0.19.0
pretty-json, v0.4.1

# Dev
No dev packages
```

**Comment by acusti** [comment]
Just published 1.8.0, which I’m hopeful resolves this issue. Please update if that isn’t the case.

**Comment by forestgagnon** [comment]
I just ran into this issue on `goto@1.8.3`

**Atom**: 1.17.2 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.11.6
**Thrown From**: [goto](https://github.com/v3ss0n/goto) package 1.8.3


### Stack Trace

Uncaught Error: ENOENT: no such file or directory, open 'undefined'

```
At fs.js:640

Error: ENOENT: no such file or directory, open 'undefined'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:168:20)
    at Object.fs.readFileSync (fs.js:508:33)
    at Object.fs.readFileSync (ELECTRON_ASAR.js:501:29)
    at SymbolIndex.module.exports.SymbolIndex.processFile (/packages/goto/lib/symbol-index.coffee:154:15)
    at SymbolIndex.module.exports.SymbolIndex.update (/packages/goto/lib/symbol-index.coffee:107:12)
    at SymbolIndex.module.exports.SymbolIndex.gotoDeclaration (/packages/goto/lib/symbol-index.coffee:127:6)
    at Object.gotoDeclaration (/packages/goto/lib/index.coffee:44:22)
    at /packages/goto/lib/index.coffee:30:31)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:1)
    at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:1
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:1)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:1)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:1)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:1)
```

### Commands

```
     -3:56.1.0 core:close (input.hidden-input)
     -1:05.0 intentions:highlight (input.hidden-input)
     -1:04.9.0 goto:declaration (input.hidden-input)
     -0:57.6.0 core:move-down (input.hidden-input)
 18x -0:57.3.0 core:move-right (input.hidden-input)
     -0:55.6.0 goto:declaration (input.hidden-input)
     -0:54.5.0 editor:consolidate-selections (input.hidden-input)
     -0:54.5.0 core:cancel (input.hidden-input)
     -0:52 intentions:highlight (input.hidden-input)
     -0:51.9.0 goto:declaration (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.29.26 
atom-clock 0.1.9 
atom-material-syntax-dark 1.0.0 
atom-ternjs 0.18.3 
atomic-chrome 0.3.0 
auto-indent 0.5.0 
autocomplete-emojis 2.5.0 
autocomplete-json 5.5.0 
autocomplete-modules 1.6.10 
autocomplete-paths 1.0.5 
autocomplete-ruby 0.2.8 
autocomplete-webgl 1.1.0 
blame 0.12.0 
busy-signal 1.4.3 
change-case 0.6.5 
character-table 0.4.3 
charcode-display 0.2.1 
color-picker 2.2.5 
color-tabs 0.1.8 
custom-title 1.0.1 
docblockr 0.10.3 
editorconfig 2.2.2 
expand-selection-to-quotes 0.11.0 
file-icons 2.1.7 
filesize 3.0.0 
flex-tool-bar 0.12.0 
git-plus 7.9.3 
git-time-machine 1.5.9 
goto 1.8.3 
goto-definition 1.3.3 
hey-pane 1.0.0 
highlight-selected 0.13.1 
indent-guide-improved 1.4.12 
intentions 1.1.2 
jumpy 3.1.3 
language-csv 1.1.2 
language-haml 0.24.5 
language-latex 1.0.0 
language-lua 0.9.11 
language-markdown 0.23.0 
language-viml 1.1.3 
linter 2.1.4 
linter-csslint 1.3.4 
linter-eslint 8.2.1 
linter-js-yaml 1.2.7 
linter-jsonlint 1.3.0 
linter-markdown 4.0.1 
linter-rubocop 2.1.1 
linter-sass-lint 1.7.4 
linter-ui-default 1.6.1 
markdown-pdf 1.5.0 
merge-conflicts 1.4.5 
minimap 4.28.2 
minimap-bookmarks 0.4.2 
minimap-codeglance 0.4.7 
minimap-cursorline 0.2.0 
minimap-highlight-selected 4.6.1 
minimap-pigments 0.2.2 
package-sync 1.1.0 
pdf-view 0.59.0 
pigments 0.39.1 
project-manager 3.3.5 
qolor 0.4.1 
react 0.16.2 
ruby-block 0.3.6 
seti-ui 1.8.0 
sort-lines 0.14.0 
tablr 1.8.3 
todo-show 1.11.0 
toggle-quotes 1.0.1 
toggler 0.3.0 
tool-bar 1.1.0 
trailing-semicolon 1.1.0 
tree-view-git-status 1.4.0 
windows-carriage-return-remover 0.1.3 
wordcount 2.10.4 
```
